### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by xlc**
Required by #46

### CODE REVIEW

File: xo/package.json [hunk 1]
```diff
@@ -65,13 +65,13 @@
     "lint": "tslint -p . --format stylish"
   },
   "dependencies": {
-    "vscode-languageclient": "^3.5.0"
+    "vscode-languageclient": "^4.1.3"
   },
   "devDependencies": {
-    "@types/node": "^6.0.52",
+    "@types/node": "^6.0.110",
```
Comment (SamVerschueren) [inline xo/package.json]: I think we can upgrade these to `^8.x.y`?

**Comment by xlc** [comment]
Updated

File: xo-server/package.json [hunk 2]
```diff
@@ -22,13 +22,13 @@
   },
   "dependencies": {
     "load-json-file": "^2.0.0",
```
Comment (SamVerschueren) [inline xo-server/package.json]: Let's upgrade `load-json-file` and `@types/load-json-file` to their latest versions as well. Just to be consistent with upgrading dependencies :).

**Comment by xlc** [comment]
Updated. `@types/load-json-file` is already latest version

**Comment by SamVerschueren** [comment]
Thanks 🎉!

**Pull request merged**